app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -78,6 +75,15 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
               grossPaymentsToDate
               calculatedTotalPaymentAmountToDate
               totalProjectValue
+              anticipatedFundingAmountPerFiscalYear {
+                edges {
+                  # eslint-disable-next-line relay/unused-fields
+                  node {
+                    anticipatedFundingAmount
+                    fiscalYear
+                  }
+                }
+              }
```

Comment:
I've done this in the form. I haven't done it in the summary because I'd have to make a custom readonly array field template that fetches the fragment